src/spacy/fortex/spacy/spacy_processors.py

```diff
@@ -95,6 +95,24 @@ def validate_spacy_configs(configs: Config):
                 "'sentence' is necessary in configs.processors for 'tokenize'."
             )
 
+    if "umls_link" in configs.processors:
+        if not (configs.medical_onto_type and configs.umls_onto_type):
+            raise ProcessorConfigError(
+                "Please specify medical and umls link ontology types!"
+            )
+
+        entry_type = get_class(configs.medical_onto_type)
+        if not isinstance(entry_type, Annotation) and not issubclass(
+            entry_type, Annotation
+        ):
+            raise ValueError("medical_onto_type must be an Annotation type.")
+
+        entry_type = get_class(configs.umls_onto_type)
+        if not isinstance(entry_type, Generics) and not issubclass(
+            entry_type, Generics
+        ):
+            raise ValueError("umls_onto_type must be a Generic type.")
```

Comment:
Similar as above.

Also, we have get these classes here, maybe we should store them in the class so we don't have to do it again.

Reply:
We are validating these configs in an independent method which is called by` initialize()`, so we will have to return these 2 classes back to the `initialize()` method for us to be able to save them to our SpacyProcessor class. Also, we don't really need these classes unless `umls_link` is enabled from config and medical entities are detected from NER. Do you think we should store them within our class anyway?

Reply:
storing the class is not crucial, just a small suggestion.